packages/patterns/src/components/create-pattern-modal.js

```diff
@@ -140,9 +143,13 @@ export function CreatePatternModalContents( {
 	 */
 	async function findOrCreateTerm( term ) {
 		try {
-			// We need to match any existing term to the correct slug to prevent duplicates, eg.
-			// the core `Headers` category uses the singular `header` as the slug.
-			const existingTerm = categoryMap.get( term );
+			const existingTerm = categoryMap.get( term.toLowerCase() );
```

Comment:
Since we don't rely on catching the error to return the term id anymore and call of `findOrCreateTerm` is in a `try catch` block(`onCreate`), I think we can also remove the `try catch` here block in this function.

Reply:
Thanks @ntsekouras, I did think about that, but there is an edge case where another user might add a category after the user has loaded the editor, in which case they won't have that category loaded and an error will be thrown. Although it would be a rare edge case I thought it was better to still handle it and return the id in this case instead of an error being thrown. What do you think?

Reply:
Oh.. okay that's an edge case alright 😄 . Since we have the code in place and doesn't affect much(being in a callback) let's keep it as if for now.